IABot/checkIfDead.php

```diff
@@ -88,33 +87,14 @@ public function checkDeadlinks( $urls ) {
 		return $returnArray;
 	}
 
+
 	/**
-	 * Function to check whether a given link is dead
-	 * @param string $url URL of the link to be checked
-	 * @return bool True if dead; False if not
-	 * @access public
-	 * @author Niharika Kohli (Niharika29)
```

Comment:
Some documentation mechanisms depend on the doc-blocks for that kind of information.  As for Authors, I can do per-file.